region/maridia/inner-pink/Colosseum.json

```diff
@@ -998,6 +1125,56 @@
         "canSpaceJumpWaterBounce"
       ]
     },
+    {
+      "link": [2, 3],
+      "name": "Come in Shinecharging, Leave With Temporary Blue (Gravity Jump HiJump)",
+      "entranceCondition": {
+        "comeInShinecharging": {
+          "length": 3,
+          "openEnd": 0
+        }
+      },
+      "requires": [
+        "canXRayTurnaround",
+        "canGravityJump",
+        "HiJump",
+        "canChainTemporaryBlue"
+      ],
+      "exitCondition": {
+        "leaveWithTemporaryBlue": {}
+      },
+      "unlocksDoors": [{"types": ["ammo"], "requires": []}]
+    },
+    {
+      "link": [2, 3],
+      "name": "Come in Shinecharging, Leave With Temporary Blue (Gravity Jump, Spring Ball Jump, Pause Buffer)",
```

Comment:
The sequence looks possible.  The morph->springballjump->unmorph is really quick and the hardest part.  The pause buffer morph is mostly difficult because it follows the springballjump sequence.  Pause buffer morph makes sense as a new tech because its not too well known.  But it could also be included in another tech, depending on if it has uses other than chainTempBlue.  Like how TAS uses a pause buffer morph to grapple fling down through bowling.

Reply:
Sounds good. In the next PR there turned out to be a lot more applications of the technique so I was adding the new tech there (`canPauseRemorphTemporaryBlue`). Went ahead and copied it over into this PR as well.